lib/features/vpc/pages/vpc_intro_page.dart

```diff
@@ -0,0 +1,83 @@
+import 'package:carousel_slider/carousel_slider.dart';
+import 'package:flutter/material.dart';
+import 'package:givt_app/features/vpc/widgets/vpc_intro_item_family.dart';
+import 'package:givt_app/features/vpc/widgets/vpc_intro_item_g4k.dart';
+import 'package:givt_app/features/vpc/widgets/vpc_intro_item_safety.dart';
+
+class VPCIntroPage extends StatefulWidget {
+  const VPCIntroPage({super.key});
+
+  @override
+  State<VPCIntroPage> createState() => _VPCIntroPageState();
+}
+
+class _VPCIntroPageState extends State<VPCIntroPage> {
+  final List<Widget> pages = const [
+    VPCIntroItemFamily(),
+    VPCIntroItemSafety(),
+    VPCIntroItemG4K(),
+  ];
+
+  var _currentPageIndex = 0;
+  final _carouselController = CarouselController();
+
+  @override
+  Widget build(BuildContext context) {
+    final size = MediaQuery.of(context).size;
+
+    return Scaffold(
+      body: Container(
+        padding: const EdgeInsets.symmetric(vertical: 35),
+        width: double.infinity,
+        child: Column(
+          children: [
+            Image.asset(
+              'assets/images/logo.png',
+              height: size.height * 0.035,
+            ),
+            Container(
+              padding: const EdgeInsets.all(20),
+              height: size.height * 0.865,
+              child: CarouselSlider(
+                carouselController: _carouselController,
+                options: CarouselOptions(
+                  height: double.infinity,
```

Comment:
viewportFraction: 1,

to not see the other slides. now they show on the sides.
![Simulator Screenshot - iPhone 11 Pro Max - 2023-07-13 at 13 59 03](https://github.com/givtnl/givt-app/assets/43034761/bedca2e1-1f0b-4754-ae48-c52fdd3e7f93)

Reply:
fixed, thanks